citadel/install_windows_src.md

```diff
@@ -0,0 +1,136 @@
+1. Open a Visual Studio Command Prompt (search for "x64 Native Tools Command Prompt for VS 2019" in the Windows search field near the Windows button). Optionally, right-click and pin to the task bar for quick access in the future.
+
+    If you did not add Conda to your `PATH` environment variable during Conda installation, you may need to navigate to the location of `condabin` in order to use the `conda` command.
+    To find `condabin`, search for "Anaconda Prompt" in the Windows search field near the Windows button, open it, run `where conda`, and look for a line containing the directory `condabin`.
+
+1. Navigate to your `condabin`, if necessary, and then create and activate a Conda environment:
+
+        conda create -n ign-ws
+        conda activate ign-ws
+
+    Once you have activate an environment, a prefix like `(ign-ws)` will be prepended to your prompt, and you can use the `conda` command outside of the `condabin` directory.
+
+    You can use `conda info --envs` to see all of your environments.
+
+    To remove an environment, use `conda env remove --name <env_name>`.
+
+1. Install dependencies
+
+        conda install cmake git vcstool curl pkg-config ^
+        colcon-common-extensions eigen freeimage gts ^
+        glib dlfcn-win32 ffmpeg ruby tinyxml2 tinyxml ^
+        protobuf urdfdom zeromq cppzmq ogre jsoncpp ^
+        libzip qt --channel conda-forge
+
+1. Navigate to where you would like to build the library, and then clone the repositories.
+
+        curl -O https://raw.githubusercontent.com/ignition-tooling/gazebodistro/master/collection-citadel.yaml
+        mkdir src
+        vcs import src < collection-citadel.yaml
+
+1. Install `ign-cmake` from `conda-forge` and disable the cloned `ign-cmake` from being built.
+
+        conda install libignition-cmake2 --channel conda-forge
+        touch src\ign-cmake\COLCON_IGNORE
+
+    This is due to a linking error currently existing within the `ign-cmake` source. See the comments [here](https://github.com/ignitionrobotics/docs/issues/96#issuecomment-742096017).
+
+**Note**
+
+You can view all available versions of a specific package with:
+```bash
+conda search libignition-<package_name>* --channel conda-forge
+```
+and view their dependencies with
+```bash
+conda search libignition-<package_name>* --channel conda-forge --info
+```
+and install a specific minor version with
+```bash
+conda install libignition-<package_name>=<major>.<minor>.<patch> --channel conda-forge
+```
+where `<major>` is the major release number, `<minor>` is the minor release number, and `<patch` is the patch release number.
+
+## Building the Ignition Libraries
+
+Once the compiler and all the sources are in place it is time to compile them.
+Start the procedure by changing into the workspace and listing the packages
```

Comment:
`changing directories`? Actually this is the first time a "workspace" is mentioned, so it'd be good to define what it's talking about, maybe at the `curl` command above.

Reply:
https://github.com/ignitionrobotics/docs/pull/126/commits/6e4b3986e8997d560d63b215ba578e4a2a173bd0#diff-f9b0c6abb8240f145d30a8a02e552a274015dfa06973edbe0b0bcb6c87224705R95